debian-stuff/katello

```diff
@@ -27,9 +27,13 @@
 import sys
 import re
 import hashlib
-
 import requests
+from six.moves.urllib.parse import quote
 
+try:
+    from ConfigParser import SafeConfigParser
+except ImportError:
+    from configparser import ConfigParser as SafeConfigParser
```

Comment:
I think this can better be as:
```python
from six.moves.configparser import SafeConfigParser
```
So it is easier for us to track this py2/3 code.